admin-api/resource_server/api-resources.md

```diff
@@ -0,0 +1,460 @@
+---
+title: API Resources
+menuTitle: API Resources
+weight: 2
+useMermaid: true
+---------
+
+# API Resources
+
+API Platform has many different ways to configure and define APIs (PHP, YAML, XML), we chose the PHP approach relying on PHP attributes to simplify and centralize the configuration in a single file which is why:
```

Comment:
same paragraph than in the previous page in the `API Resource classes` paragraph?

Reply:
You're right I'll modify the previous page with a link to this one